lib/Infer/ConstantSynthesis.cpp

```diff
@@ -421,16 +421,11 @@ ConstantSynthesis::synthesize(SMTLIBSolver *SMTSolver,
       }
     }
 
-    BlockPCs BPCsCopy;
-    std::vector<InstMapping> PCsCopy;
-    separateBlockPCs(BPCs, BPCsCopy, InstCache, BlockCache, IC, &ConstMap, false);
```

Comment:
After this patch both functions (separateBlockPCs and separatePCs) are no longer be used. Their implementation are perserved because there may be some potential G by S improvement that requires these routines in the future.